lib/plugins/package/lib/packageService.js

```diff
@@ -93,7 +93,8 @@ module.exports = {
     }
 
     // use the artifact in service config if provided
-    if (this.serverless.service.package.artifact) {
+    // and if the function is not set to be packaged individually
+    if (this.serverless.service.package.artifact && !funcPackageConfig.individually) {
```

Comment:
Use `lodash` like `_.isEmpty(funcPackageConfig.individually)` for consistancy of other places.

Reply:
I don't think that that is correct.
`funcPackageConfig.individually` is a boolean so `_.isEmpty(...)` will return true for all cases (true, false, undefined, null, etc.).
I searched the code base and couldn't find any example of using `_.isEmpty` in on boolean values.

Please correct me if I have misunderstood anything.

Reply:
Make sense 👍 I was not correct.
Thank you for the response and investigating about `_.isEmpty` 🥇